EIPS/eip-4973.md

```diff
@@ -0,0 +1,127 @@
+---
+eip: 4973
+title: Non-Transferrable Non-Fungible Account-bound Tokens (formerly Soulbound Tokens or "Badges")
+description: A standard interface for non-transferrable non-fungible tokens, also known as "account-bound" or "soulbound tokens" or "badges".
+author: Nicola Greco, Tim Daubenschütz (@TimDaub)
+discussions-to: https://ethereum-magicians.org/t/eip-4973-non-transferrable-non-fungible-tokens-soulbound-tokens-or-badges/8825
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-04-01
+requires: 165, 721
+---
+
+## Abstract
+
+Proposes a standard API for account-bound Tokens (ACT), also known as Badges, within smart contracts. An ACT is a non-fungible token bound to a single account, and cannot be transferred between accounts. This EIP defines basic functionality to gift, mint, and track ACTs.
```

Comment:
Perhaps worth excluding the reference to these as "Badges" and just call them "Account Bound Tokens" for the sake of ubiquity this may impose. 

Also, why not abbreviate to ABT? 

```suggestion
Proposes a standard API for account-bound Tokens (ACT), within smart contracts. An ACT is a non-fungible token bound to a single account, and cannot be transferred between accounts. This EIP defines basic functionality to gift, mint, and track ACTs.
```

Reply:
> Also, why not abbreviate to ABT?

Not sure why I used the term "ACT", it makes no sense and so I'm shipping an update where the abbreviation is "ABT".

Reply:
"Account" strikes me as a misrepresentation because they are _key_-bound; the term "account" doesn't necessarily imply this.